Lib/pdb.py

```diff
@@ -1238,6 +1238,8 @@ def do_args(self, arg):
     def do_retval(self, arg):
         """retval
         Print the return value for the last return of a function.
+        Alternatively, the return value can be accessed with the local
+        variable ``__return__``
```

Comment:
Actually this sentence fits better in the documentation here: https://docs.python.org/3/library/pdb.html#pdbcommand-retval

For the docstring it's not really appropriate to point to an alternative.